pos_pricelist/static/src/js/widgets.js

```diff
@@ -63,5 +65,24 @@ function pos_pricelist_widgets(instance, module) {
             this.pos.pricelist_engine.update_products_ui(customer);
         }
     });
+
+    module.PosBaseWidget.include({
+        format_pr: function(amount, precision) {
+            console.log('format_pr: amount = ' + amount)
+            // Do not call _super because no addon or XML is using this method
```

Comment:
Yes it is defined here: https://github.com/odoo/odoo/blob/8.0/addons/point_of_sale/static/src/js/widget_base.js#L47 but it not used by anyone. Looks like a deprecated code. For this improvement is cool because I need to show price with currency configuration (decimals) but without currency symbol.